source/using_api/odata/odata_basics.rst

```diff
@@ -248,7 +250,7 @@ GET |url6|_
 .. _url6: http://public.opendatasoft.com/api/odata/baby_names_nc_2013?$top=2
 
 .. code-block:: json
```

Comment:
the JSON code-block does exists, but it will raise warnings if the content is not pure json (double quotes, no comments etc.). This one was fine.